opentelemetry-api/opentelemetry/distributedcontext/__init__.py

```diff
@@ -1 +1,13 @@
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
Currently I keep the `__init__.py` file here just to keep the folder structure (otherwise the empty folder will get deleted).
Going forward, we need to remove `__init__.py` if this folder is intended for a namespace package.

Reply:
I thought about this and figured that probably we don't need all folders to be namespace, for example, the context folder in the API package will contain the actual implementation.
In this case, putting .gitkeep seems to be overkill?
I don't have strong opinion here, would be happy to make changes if it make sense.

Reply:
True, for packages that we don't want to be namespace packages, a normal empty init.py would be best.

Reply:
Changed to empty `__init__.py` file.

Reply:
Ah, by "empty" I didn't mean to object to the license comment -- either way is fine with me.

Reply:
I see, I've put the license comment back.